app/src/main/java/org/simple/clinic/cvdrisk/CVDRiskCalculator.kt

```diff
@@ -0,0 +1,71 @@
+package org.simple.clinic.cvdrisk
+
+import org.simple.clinic.patient.Answer
+import org.simple.clinic.patient.Gender
+
+object CVDRiskCalculator {
+
+  fun calculateCvdRisk(
+      cvdRiskData: CVDRiskCalculationSheet?,
```

Comment:
If I understand correctly, this sheet data won't change from patient to patient right? Why don't we pass this as dependency to this class and use it here instead of passing it to function everywhere we call it? 🤔

Reply:
Done.